tests/package.json

```diff
@@ -15,8 +15,12 @@
     "test:coverage": "npm t -- --coverage",
     "test:ci": "npm t -- --ci --colors"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@babel/runtime": "^7.7.2"
```

Comment:
Is this needed? Wouldn't it be a dependency of axe-puppeteer?

Reply:
Without it, when I run `npm test` I get  `Cannot find module '@babel/runtime/helpers/interopRequireDefault' from 'index.js'`

Reply:
Does it tell you which index.js? Sounds like a bug on their end, if they need that dependency it should be included in their pacakge.json.
Could you check the issue trackers for the two new dependencies to see if it has been reported already?
In the meantime, we should move this to dev dependencies, hopefully we can remove it when it get's fixed.

Reply:
I can't find an issue for it, but it's from this package https://github.com/WordPress/gutenberg/tree/master/packages/jest-puppeteer-axe which is checking against index.js for @babel/runtime/helpers/interopRequireDefault  in the node module when it's built

Reply:
I don't see any babel imports in https://github.com/WordPress/gutenberg/blob/master/packages/jest-puppeteer-axe/src/index.js
How did you find out this is the one causing the issue? Is it one of its dependencies?

Reply:
When I build it I get the first lines in `build-module/index.js`

```
import _regeneratorRuntime from "@babel/runtime/regenerator";
import _asyncToGenerator from "@babel/runtime/helpers/esm/asyncToGenerator";
```
and in `build/index.js`

```
var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _regenerator = _interopRequireDefault(require("@babel/runtime/regenerator"));

var _asyncToGenerator2 = _interopRequireDefault(require("@babel/runtime/helpers/asyncToGenerator"));

var _axePuppeteer = _interopRequireDefault(require("axe-puppeteer"));

```

Reply:
Okay, so that is only in the packaged code shipped by npm.
Could you open an issue in the repo? Seeing it is a very small project inside a very large project (Gutenberg), it might not be a priority, but we should report it anyway.

Reply:
I think this might be the broader issue https://github.com/WordPress/gutenberg/issues/14373 What do you think?

Reply:
Yeah, that is definitely related to this issue, but it was raised in March and it hasn't had any activity since then. Would you mind creating a PR like this one on that repo? Maybe that will get merged quickly.
https://github.com/WordPress/gutenberg/pull/14374

Reply:
Have done so here https://github.com/WordPress/gutenberg/pull/18626

Reply:
Thanks Mark!